Dockerfile

```diff
@@ -68,38 +76,38 @@ RUN set -x \
     && grep " node-v$NODE_VERSION-linux-x64.tar.gz\$" SHASUMS256.txt.asc | sha256sum -c - \
     && tar -xzf "node-v$NODE_VERSION-linux-x64.tar.gz" -C /usr/local --strip-components=1 \
     && rm -r "$GNUPGHOME" "node-v$NODE_VERSION-linux-x64.tar.gz" SHASUMS256.txt.asc \
-    && apt-get purge -y --auto-remove wget
-
-RUN npm install -g yarn
+    && apt-get purge -y --auto-remove wget \
+    && npm install -g yarn@$YARN_VERSION \
```

Comment:
Does it just not use a cache anymore? I can investigate, but yeah, I had to use `--force`. But I find it weird to say it doesn't _need_ it if we strictly don't want it since there's no value in caching here at all.